### ISSUE TITLE
[Bug?]: @yarnpkg/builder 3.0.0 package does not contain lib folder

### ISSUE THREAD

**Issue by ksm2** [yarnpkg/berry#3212]
### Self-service

- [ ] I'd be willing to implement a fix

### Describe the bug

While trying to build a plugin for Yarn 3, I realised that the `@yarnpkg/builder` is missing the lib folder containing the binary. This causes the following error when running `builder build plugin`:

```
/path/to/my/project/.pnp.cjs:17892
    throw firstError;
    ^

Error: Qualified path resolution failed - none of those files can be found on the disk.

Source path: /path/to/my/project/.yarn/__virtual__/@yarnpkg-builder-virtual-ae32286188/0/cache/@yarnpkg-builder-npm-3.0.0-a7c51e0be3-cc2b64b2c5.zip/node_modules/@yarnpkg/builder/lib/cli.js
Not found: /path/to/my/project/.yarn/__virtual__/@yarnpkg-builder-virtual-ae32286188/0/cache/@yarnpkg-builder-npm-3.0.0-a7c51e0be3-cc2b64b2c5.zip/node_modules/@yarnpkg/builder/lib/cli.js
Not found: /path/to/my/project/.yarn/__virtual__/@yarnpkg-builder-virtual-ae32286188/0/cache/@yarnpkg-builder-npm-3.0.0-a7c51e0be3-cc2b64b2c5.zip/node_modules/@yarnpkg/builder/lib/cli.js.js
Not found: /path/to/my/project/.yarn/__virtual__/@yarnpkg-builder-virtual-ae32286188/0/cache/@yarnpkg-builder-npm-3.0.0-a7c51e0be3-cc2b64b2c5.zip/node_modules/@yarnpkg/builder/lib/cli.js.json
Not found: /path/to/my/project/.yarn/__virtual__/@yarnpkg-builder-virtual-ae32286188/0/cache/@yarnpkg-builder-npm-3.0.0-a7c51e0be3-cc2b64b2c5.zip/node_modules/@yarnpkg/builder/lib/cli.js.node

    at Function.external_module_.Module._resolveFilename (/path/to/my/project/.pnp.cjs:17891:55)
    at Function.external_module_.Module._load (/path/to/my/project/.pnp.cjs:17690:48)
```

Could you try to re-release the package with the binary included?

### To reproduce

```
yarn add @yarnpkg/builder
yarn builder build plugin
```

### Environment

  System:
    OS: Linux 5.10 Arch Linux
    CPU: (4) x64 Intel(R) Core(TM) i7-6600U CPU @ 2.60GHz
  Binaries:
    Node: 16.4.0 - /tmp/xfs-b1ad1139/node
    Yarn: 3.0.0 - /tmp/xfs-b1ad1139/yarn
    npm: 7.17.0 - /usr/bin/npm

**Comment by arcanis** [comment]
I just released 3.0.1 which should fix the problem.

**Comment by ksm2** [comment]
Thanks for picking this up so quickly, @arcanis! I can confirm it's working fine now.
